app/src/components/cards/BlockDetailsCard/BlockDetailsCard.tsx

```diff
@@ -55,179 +54,203 @@ export const BlockDetailsCard: React.FC<BlockDetailsCardProps> = ({
+            </DetailDataValue>
+          </li>
+        </DetailDataWrapper>
+
+        <Spacer height="2.5rem" />
+
+        <DetailDataRowWrapper>
+          <li>
+            <DetailDataLabel>{t('raw-data')}</DetailDataLabel>
+            <DetailDataValue>
+              <RawData rawData={rawBlock} />
+            </DetailDataValue>
+          </li>
+          <li>
+            <DetailDataLabel>{t('deploys')}</DetailDataLabel>
+            <DetailDataValue isLargeText>
+              {withSkeletonLoading(
+                block?.body.deploy_hashes?.length ? (
+                  <ul>
+                    {block?.body.deploy_hashes?.map(deployHash => (
+                      <li key={deployHash}>
+                        <a
+                          href={`/deploy/${deployHash}`}
+                          style={{ color: '#2230f0' }}>
+                          <Hash
+                            alwaysTruncate
+                            hash={deployHash ?? hashPlaceholder}
+                          />
+                        </a>
+                      </li>
+                    ))}
+                  </ul>
+                ) : (
+                  t('no-deploys')
+                ),
+                isLoading,
+                { width: 150 },
+              )}
+            </DetailDataValue>
+          </li>
+          <li>
+            <DetailDataLabel>{t('transfers')}</DetailDataLabel>
+            <DetailDataValue isLargeText>
+              {withSkeletonLoading(
+                block?.body.transfer_hashes?.length ? (
+                  <ul>
+                    {block?.body.transfer_hashes?.map(transferHash => (
+                      <li key={transferHash}>
+                        <a
+                          href={`/deploy/${transferHash}`}
+                          style={{ color: '#2230f0' }}>
+                          <Hash
+                            alwaysTruncate
+                            hash={transferHash ?? hashPlaceholder}
+                          />
+                        </a>
+                      </li>
+                    ))}
+                  </ul>
+                ) : (
+                  t('no-transfers')
+                ),
+                isLoading,
+                { width: 150 },
+              )}
+            </DetailDataValue>
+          </li>
+        </DetailDataRowWrapper>
+      </InfoCard>
+    </>
   );
 };
 
-const AccountHeading = styled(Heading)`
-  font-size: 1.125rem;
-  font-weight: 500;
-  margin-bottom: 1rem;
-`;
-
 const HashWrapper = styled.div`
   display: flex;
   flex-direction: column;
 `;
 
-const HashHeading = styled(GradientHeading)<{ isTruncated: boolean }>`
-  font-weight: ${fontWeight.extraBold};
+const HashHeading = styled(Heading)<{ isTruncated: boolean }>`
+  font-weight: ${fontWeight.medium};
   display: inline;
   margin: 0;
-  width: ${({ isTruncated }) => (isTruncated ? '10%' : '100%')};
+  width: ${({ isTruncated }) => (isTruncated ? '40%' : '75vw')};
```

Comment:
Check out the `ch` css unit.  It's quite good at being consistent at font or character changes

Reply:
Oh yeah I always forget about that guy. It actually works quite well with vw so I think I'll just go with that 👍

Reply:
Okay sounds good